### PR TITLE
Issue #306: Failed to generate download URL for artifact : No module …

### PR DESCRIPTION
**Pull request by sytianhe**

### CODE REVIEW

File: build/ax/build/devops.py [hunk 1]
```diff
@@ -111,12 +111,13 @@ def _copy_source_code(self, container_build_path):
         run_cmd('cp -LR {} {}/src'.format(PYTHON_COMMON_PATH, docker_build_dir))
         run_cmd('cp -LR {}/devops/requirements {}'.format(SRC_PATH, docker_build_dir))
 
-        if not re.search('devopsbuilder', container_build_path):
-            # DevOps source code is not protected (for the time being). Platform code is.
-            # Do not copy platform code into devops containers to maintain this protection
-            # and prevent exposure of our platform source to customers.
-            shutil.rmtree("{}/src/ax/platform".format(docker_build_dir))
-            shutil.rmtree("{}/src/ax/platform_client".format(docker_build_dir))
+        # Issue #306 devops component requires platform_client. This is only needed from release 1.0.1
```
Comment (shrinandj) [inline build/ax/build/devops.py]: Why not delete this, given that there are no known reasons to ever bring it back?

Comment (sytianhe) [reply build/ax/build/devops.py]: I feel this is a specific patch and would like to give a heads up ...

Comment (shrinandj) [reply build/ax/build/devops.py]: Not sure who the heads up is for and who/what will delete this dead code eventually. Let's remove this and check-in.

Comment (jessesuen) [reply build/ax/build/devops.py]: As the original author of this code, please go ahead delete it since it no longer makes sense without pyinstaller.

**Pull request merged**